FreeRTOS-Plus/VisualStudio_StaticProjects/MbedTLS/mbedtls_freertos_port.c

```diff
@@ -45,6 +39,13 @@
 
 #include "mbedtls_freertos_port.h"
 
+
+#include "FreeRTOSConfig.h"
+
+/* FreeRTOS includes. */
+#include "FreeRTOS.h"
+#include "semphr.h"
+
```

Comment:
Curious why this change was required?

Reply:
This partly solved/worked around the `winsock2.h` issue that was created with the new version of MbedTLS

I believe at this point it could be reverted if you want

Reply:
I think there was another reason I had wanted to include the MbedTLS config first? But I can't remember if there was an actual reason to or not at this point

Reply:
I think we can keep this change. I was just curious why would we need to do this and to understand whether FreeRTOS headers were including some files which did not work well with the mbedtls files.

Reply:
I actually went ahead and reverted it - I don't think this change is actually needed, so would rather keep the file as is.